cypress-tests/cypress/e2e/PaymentUtils/WellsFargo.js

```diff
@@ -0,0 +1,492 @@
+const successfulNo3DSCardDetails = {
+    card_number: "4242424242424242",
+    card_exp_month: "01",
+    card_exp_year: "25",
+    card_holder_name: "joseph Doe",
+    card_cvc: "123",
+};
+
+const successfulThreeDSTestCardDetails = {
+    card_number: "4000000000001091",
+    card_exp_month: "01",
+    card_exp_year: "25",
+    card_holder_name: "joseph Doe",
+    card_cvc: "123",
+};
+
+const singleUseMandateData = {
+    customer_acceptance: {
+        acceptance_type: "offline",
+        accepted_at: "1963-05-03T04:07:52.723Z",
+        online: {
+            ip_address: "125.0.0.1",
+            user_agent: "amet irure esse",
+        },
+    },
+    mandate_type: {
+        single_use: {
+            amount: 8000,
+            currency: "USD",
+        },
+    },
+};
+
+const multiUseMandateData = {
+    customer_acceptance: {
+        acceptance_type: "offline",
+        accepted_at: "1963-05-03T04:07:52.723Z",
+        online: {
+            ip_address: "125.0.0.1",
+            user_agent: "amet irure esse",
+        },
+    },
+    mandate_type: {
+        multi_use: {
+            amount: 8000,
+            currency: "USD",
+        },
+    },
+};
+
+export const connectorDetails = {
+    card_pm: {
+        PaymentIntent: {
+            Request: {
+                payment_method_data: {
```

Comment:
please add `payment_method: "card",` to all card exchanges (`Request`) as shown below. I've this [PR](https://github.com/juspay/hyperswitch/pull/5481) enforcing that change.

```suggestion
                payment_method: "card",
                payment_method_data: {
```

Reply:
Changed